lib/iris/tests/integration/test_regridding.py

```diff
@@ -30,7 +30,10 @@
 from iris.coord_systems import GeogCS
 from iris.coords import DimCoord
 from iris.cube import Cube
-from iris.tests.stock import global_pp
+from iris.tests.stock import global_pp, simple_3d
+from iris.analysis import UnstructuredNearest
+
+iris.FUTURE.netcdf_promote = True
```

Comment:
i think this is causing a problem with your tests
it is a global switch, so can have wide ranging effects.  to use it within a test, use it as a context manager
```
with iris.FUTURE.netcdf_promote = True:
    do_stuff()
```
or somesuch